docker/docker-compose.dev.yml

```diff
@@ -138,5 +138,38 @@ services:
       API_KEYS: bubatzbieber6942096420
       ACCESS_TOKEN_BLACKLISTING: 'true'
 
+  oidc-server-mock:
+    container_name: oidc-server-mock
+    image: ghcr.io/soluto/oidc-server-mock:latest
```

Comment:
This is https://github.com/Soluto/oidc-server-mock, which uses https://duendesoftware.com/products/identityserver

The license of this is a bit shitty, but should be okay since we only use it for local development? Do we need to ask lawyers?

Reply:
> You do not need a license for evaluation, development, test environments, or personal projects. Simply ignore the startup warning message; it will not constrain your server application in any way.
>
> A good place to start in our documentation are our [hands-on labs](https://docs.duendesoftware.com/quickstarts/).

Source https://duendesoftware.com/products/communityedition

Reply:
I think it's fine as long as we use it for development only